addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1597,7 +1597,7 @@ export function isUnbreakable(node) {
 
 export function isUnremovable(node) {
     return (
-        (node.nodeType !== Node.ELEMENT_NODE && node.nodeType !== Node.TEXT_NODE) ||
+        (node.nodeName !== "#comment" && node.nodeType !== Node.ELEMENT_NODE && node.nodeType !== Node.TEXT_NODE) ||
```

Comment:
Thanks for the PR! Why not continue with the same style as the rest of the checks though? Why not `node.nodeType !== Node.COMMENT_NODE` ?

Reply:
Thanks for your tips. Changes done :)